### PR TITLE
CDH-3347: Adaptation of inventory offloading of DataHub

### PR DESCRIPTION
**Pull request by cheinz-sag**
Hi Beate and Marlene,

can you pls review and merge to:
- 10.15 (branch was derived from r10.15)
- 10.13
- 10.11

For 10.16 the update will be included as part of PR #747.

Thanks!

### CODE REVIEW

File: content/datahub/working-with-datahub-bundle/offloading-base-collections.md [hunk 1]
```diff
@@ -35,30 +35,30 @@ The alarm collection keeps track of alarms which have been raised. During offloa
...
+The alarms collection keeps track of alarms. An alarm may change its status over time. The alarms collection also 
+supports updates to incorporate these changes. Therefore an offloading pipeline for the alarms collection encompasses 
+additional steps:
+
+1. Offload those entries of the alarms collection that were added or updated since the last offload. They are offloaded 
+with the above mentioned standard schema into the target table of the data lake.
+2. Additional views on the target table are defined in the tenant's space in Dremio. Their names are composed as 
+follows: target table name plus *_all*, *_latest*, or *_c8y_cdh_latest_materialized* respectively. The following 
+examples use "alarms" as target table name:
+    * **alarms_all**: A view with the updates between two offloading executions, not including the intermediate updates.
```
Comment (BeateRixen) [inline content/datahub/working-with-datahub-bundle/offloading-base-collections.md:51]:
```suggestion
    * **alarms_all** - A view with the updates between two offloading executions, not including the intermediate updates. 
```

File: content/datahub/working-with-datahub-bundle/offloading-base-collections.md [hunk 2]
```diff
@@ -35,30 +35,30 @@ The alarm collection keeps track of alarms which have been raised. During offloa
...
+with the above mentioned standard schema into the target table of the data lake.
+2. Additional views on the target table are defined in the tenant's space in Dremio. Their names are composed as 
+follows: target table name plus *_all*, *_latest*, or *_c8y_cdh_latest_materialized* respectively. The following 
+examples use "alarms" as target table name:
+    * **alarms_all**: A view with the updates between two offloading executions, not including the intermediate updates. 
+    For example, after the first offloading execution, the status of an alarm is ACTIVE. Then it changes its status 
+    from ACTIVE to INACTIVE and afterwards back to ACTIVE. When the next offloading is executed, it will persist the 
+    latest status ACTIVE, but not the intermediate status INACTIVE (because it happened between two offloading runs and 
+    thus is not seen by {{< product-c8y-iot >}} DataHub).
     * **alarms_latest**: A view with the latest status of all alarms, with all previous transitions being discarded.
```
Comment (BeateRixen) [inline content/datahub/working-with-datahub-bundle/offloading-base-collections.md:56]:
```suggestion
    * **alarms_latest** - A view with the latest status of all alarms, with all previous transitions being discarded.
```

File: content/datahub/working-with-datahub-bundle/offloading-base-collections.md [hunk 3]
```diff
@@ -35,30 +35,30 @@ The alarm collection keeps track of alarms which have been raised. During offloa
...
-    * **alarms_c8y_cdh_latest_materialized**: An optional view which materializes the **alarms_latest** view if the 
-    offloading configuration has view materialization enabled.
+    * **alarms_c8y_cdh_latest_materialized**: An optional view which materializes the **alarms_latest** view if the
```
Comment (BeateRixen) [inline content/datahub/working-with-datahub-bundle/offloading-base-collections.md:57]:
```suggestion
    * **alarms_c8y_cdh_latest_materialized** - An optional view which materializes the **alarms_latest** view if the 
```

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Comment by BeateRixen** [comment]
> Hi Beate and Marlene,
> 
> can you pls review and merge to:
> 
> * 10.15 (branch was derived from r10.15)
> * 10.13
> * 10.11
> 
> For 10.16 the update will be included as part of PR #747.
> 
> Thanks!

@cheinz-sag I assume this should go into 10.14 as well.

**Pull request merged**